example/performance/README.md

```diff
@@ -0,0 +1,106 @@
+make them the same size!
+
+First, we tell Palisade where its bootstrap configuration file is:
+```bash
+export PALISADE_REST_CONFIG_PATH=./example/example-model/src/main/resources/configRest.json
+```
+
+Now, to create some test data in a `perf_test` directory:
+
+```bash
+mkdir perf_test
+./example/deployment/bash-scripts/perf.sh 1000 10000 perf_test
+```
+
+This will produce output similar to:
+```bash
+example.perf.Perf INFO  - Going to create 1000 records in file employee_small.avro and 10000 records in file employee_large.avro in sub-directory
+example.perf.Perf INFO  - Creation tasks submitted...
+example.perf.Perf INFO  - Small file written successfully true
+example.perf.Perf INFO  - Large file written successfully true
+example.perf.Perf INFO  - Copying small file
+example.perf.Perf INFO  - Copying large file
+```
+
+We must then instruct Palisade to create some compliance policies regarding these files:
+```bash
+./example/deployment/bash-scripts/perf.sh policy perf_test
+```
+This then produces:
+```bash
+example.perf.Perf INFO  - Specified path perf_test has been normalised to file:///Users/username/Palisade/perf_test/
+example.perf.Perf INFO  - Security policy has been set for file:///Users/username/Palisade/perf_test/employee_small.avro: true
+example.perf.Perf INFO  - Security policy has been set for file:///Users/username/Palisade/perf_test/large/employee_large.avro: true
+example.perf.Perf INFO  - Security policy has been set for file:///Users/username/Palisade/perf_test/employee_small-nopolicy.avro: true
+example.perf.Perf INFO  - Security policy has been set for file:///Users/username/Palisade/perf_test/large/employee_large-nopolicy.avro: true
+```
+
+### Running
+
+Now we can run the performance tests:
+```bash
+./example/deployment/bash-scripts/perf.sh run perf_test 2 5
+```
+The first number is the number of "dry runs" to perform; these are tests where the results are discarded to allow JVM optimisation/class loading
+occur and not interfere with the results. The second number is the number of live runs to perform.
+
+Depending on the size of the files you created, this may take some time to run. Output will be similar to the following:
+```bash
+example.perf.Perf INFO  - Specified path perf_test has been normalised to file:///Users/username/Palisade/perf_test/
+example.perf.Perf INFO  - Starting dry runs
+Starting test read_large_native:10000
+
+<<< output snipped >>>
+
+example.perf.Perf INFO  - Starting live tests
+Starting test read_large_native:10000
+
+<<< output snipped >>>
+
+All times in seconds.
+
+Test                              # trials         Min         Max        Mean    Std.dev.         25%         50%         75%         99%        Norm
+read_small_native                    5.000       0.064       0.081       0.072       0.007       0.066       0.069       0.078       0.081       1.000
```

Comment:
it would be worth explaining the format of the test names in an intro paragraph

Reply:
Done